### ISSUE TITLE
No error message for unchecked terms & conditions in registration form (accessibility)

### ISSUE THREAD

**Issue by M-Behr** [intershop/intershop-pwa#1792]
**Actual Behavior**

No error message is visible if term & conditions checkbox is unchecked.

**Expected Behavior**

Display the error message "You must agree to the terms to continue." and make sure that the screenreader is notified of the issue. See checkout.
If a user submits a form without checking a required checkbox, the error message should be clearly visible and associated with the checkbox. This helps visual and screen reader users understand what went wrong.

**Steps to Reproduce the Bug**

1. Go to 'registration page'
2. Scroll down
3. Click 'Submit'
4. Scroll down to 'terms & conditions' checkbox

**Additional Context, like Screenshots, Log File Snippets etc.**

![Image](https://github.com/user-attachments/assets/2d48dfbc-09a2-4ced-98c3-9cfd009ce508)
